src/leo-client-app/src/components/SatellitesOfInterest.tsx

```diff
@@ -22,7 +33,18 @@ type SatelliteDetails = {
 
 const SatellitesOfInterest = ({ userId }: Props) => {
   const [satellites, setSatellites] = useState<SatelliteDetails[]>([]);
+  const [open, setOpen] = useState(false);
+  const [satelliteName, setSatelliteName] = useState('');
+  const [noradId, setNoradId] = useState('');
 
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+  
```

Comment:
can just be handleClick = () => {setOpen(!open)}, or something like that, make it 1 function